app/controllers/api/v2/exchange_rates/monthly_exchange_rates_controller.rb

```diff
@@ -0,0 +1,40 @@
+module Api
+  module V2
+    module ExchangeRates
+      class MonthlyExchangeRatesController < ApiController
+        def show
+          if month && year
+            render json: serialized_exchange_rates_list
+          else
+            render json: { "errors": [{ "detail": 'not found' }] }, status: :not_found
```

Comment:
I think we can just raise an exception to be more idiomatic with the rest of the controllers?

Reply:
Okay this is a can of  :worm: :worm: